### ISSUE TITLE
train my dataset

### ISSUE THREAD

**Issue by alirezarmzn4** [bytedance/LatentSync#74]
I want to train on my own dataset but I don't know where to start. If possible, please help me.

**Comment by zhjygit** [comment]
训练集是使用lwr1000那个吗？

**Comment by alirezarmzn4** [comment]
> 训练集是使用lwr1000那个吗？

No, I am using my own dataset in Persian. Please tell me the pre-processing and training steps. What was your accuracy? What about the loss?

**Comment by zhjygit** [comment]
I just downloaded the datasheet more than 100GB.I will have a try.

**Comment by sultanchamberlain** [comment]
[zhjygit](https://github.com/zhjygit)

Did you have success training with the LR3 dataset?

**Comment by zhjygit** [comment]
> [zhjygit](https://bgithub.xyz/zhjygit)
> 
> Did you have success training with the LR3 dataset?

no,What I downloaded was actually a CMLR dataset and each video didn't meet the preprocessing requirements in terms of length and clarity.

**Comment by zhjygit** [comment]
> > 训练集是使用lwr1000那个吗？
> 
> No, I am using my own dataset in Persian. Please tell me the pre-processing and training steps. What was your accuracy? What about the loss?
What I downloaded was actually a CMLR dataset and each video didn't meet the preprocessing requirements in terms of length and clarity.
Where did your dataset be downloaded from, and does it meet the preprocessing script requirements? Can you share the process and steps?

**Comment by theAbyssOfTime2004** [comment]
> no,What I downloaded was actually a CMLR dataset and each video didn't meet the preprocessing requirements in terms of length and clarity.

How do i know the preprocessing requirements
